spec/services/csv/lettings_log_csv_service_spec.rb

```diff
@@ -1,6 +1,16 @@
 require "rails_helper"
+require "rake"
 
 RSpec.describe Csv::LettingsLogCsvService do
+  subject(:task) { Rake::Task["data_import:add_variable_definitions"] }
+
+  before do
+    Rake.application.rake_require("tasks/log_variable_definitions")
+    Rake::Task.define_task(:environment)
+    task.reenable
+    task.invoke("spec/fixtures/variable_definitions")
```

Comment:
- I could merge the definitions into one file. 
- The tests that check the rows of the CSVs against those in the spec/fixtures/files directory need all definitions, we have all the attributes so need all the definitions. Unless, we remove definitions from these files and leave them blank for ones we know will not be in the test db. It also does test against definitions of different user types because of those files, some being 'support' some being 'user'.